pages/services/spark/2.8.0-2.4.0/release-notes/index.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 10
+excerpt: Release notes for DC/OS Apache Spark 2.8.0-2.4.0
+featureMaturity:
+render: mustache
+model: /services/spark/data.yml
+---
+
+## Version Spark and Spark history 2.8.0-2.4.0
+
+### New features
+- Mesos checkpointing support for Spark Drivers submitted to Dispatcher
```

Comment:
can you (or help from Docs team) add an anchor tag to the fault-tolerance section of that page so users can quickly jump to the information?

Reply:
added